kpi/management/commands/delete_base_command.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import sys
+
+from django.core.management.base import BaseCommand as DjangoBaseCommand
+from django.db import transaction, connection
+
+
+class BaseCommand(DjangoBaseCommand):
```

Comment:
:heart: the reusable code, but :broken_heart: using the name of a Django built-in :grin: